src/Ocelot/DependencyInjection/MergeOcelotJson.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Ocelot.DependencyInjection;
+
+public enum MergeOcelotJson
```

Comment:
No XML docs for public enumeration!
Each option should be explained to a developer.